docs/api/IoT.js-API-BLE.md

```diff
@@ -104,4 +91,101 @@ ble.on('advertisingStart', function(error) {
     ]);
   }
 });
-```
\ No newline at end of file
+```
+
+
+## Class: Descriptor
+
+Descriptors are defined attributes that describe a characteristic value.
+
+### new Descriptor(options)
+* `options` {Object}
+  * `uuid:` {string} A Universally Unique ID (UUID) is a 16 or 128-bit value used to identify the type of every attribute.
```

Comment:
Is this a hex string? Would worth a description in the main text part (still missing).